src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -432,6 +432,21 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         manager.register_pass<ov::pass::BroadcastElementwiseFusion>();
         manager.register_pass<ov::pass::BroadcastTransition>();
 
+        const bool keep_precision_sensitive_in_fp32_1 = true;
+        const bool convert_input_output_precision = false;
+        const bool store_original_precision_as_rt_attribute = true;
+
+        manager.register_pass<ov::pass::KeepDequantizationPrecision>(
+            ov::element::TypeVector{ov::element::i32, ov::element::u32, ov::element::u16});
+
+        manager.register_pass<ov::pass::ConvertPrecision>(fp_convert_precision_map,
```

Comment:
What about `manager.register_pass<ov::pass::KeepConstPrecision>(supported_woq_types, !device_info.supports_immad);` call? I believe placing this call after ConvertPrecision and CommonOptimizations might lead to the folding of zero-point calculation subgraphs. As result, data type of zero points could be changed to f16, which is not expected by oneDNN

Reply:
I moved up `ConvertPrecision` that is for floating point types not for integer types. I'll check if it has any performance and accuracy impact. Thank you.

Reply:
CommonOptimizations internally calls ConstantFolding as well, and if zero-point constants are not marked, I assume they might still be folded - please double-check it